docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>Instead of "import ott", change to "from ott.tools import plot" and later use "plot.Plot".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>